pkg/node/node.go

```diff
@@ -611,7 +611,7 @@ func (n Node) fetchKubernetesNode(nodeName string) (*corev1.Node, error) {
 	listOptions := metav1.ListOptions{LabelSelector: labels.Set(labelSelector.MatchLabels).String()}
 	matchingNodes, err := n.drainHelper.Client.CoreV1().Nodes().List(context.TODO(), listOptions)
 	if err != nil || len(matchingNodes.Items) == 0 {
-		log.Err(err).Msgf("Error when trying to list Nodes w/ label, falling back to direct Get lookup of node")
+		log.Warn().Msgf("Unable to list Nodes w/ label, falling back to direct Get lookup of node")
```

Comment:
Should the output include the `err`?

```suggestion
		log.Warn().Err(err).Msgf("Unable to list Nodes w/ label, falling back to direct Get lookup of node")
```

Reply:
From the comments, I believe it would be more clear to keep it as a Warning and not include Error in the output!